src/utils/index.js

```diff
@@ -222,6 +222,19 @@ export const isRunningLocally = () => {
   return privateIpRanges.some((regex) => regex.test(hostname));
 };
 
+/**
+ * Sanitized pathname with experiment-tracking string and check if path containing trailing slash
```

Comment:
maybe this now should mention about github pages for shareable viz rather than experiment tracking?